string location
string forecastTime
string description
int32 celsius
string precipitation
string humidity
string wind
